setup.py

```diff
@@ -0,0 +1,42 @@
+from setuptools import find_packages, setup
+
+with open("README.md", "r") as file:
+    long_description = file.read()
+
+setup(
+    name="django-polaris-wyre",
+    description="A Django Polaris extension that adds Wyre's Custodial Wallet support",
+    long_description=long_description,
+    version="0.1.0",
```

Comment:
What do you think about using some of these methods here ?
https://packaging.python.org/guides/single-sourcing-package-version/

Reply:
Sounds good! Do you think that we should add the version (`__version__`) in the `__init__` file of the package?

Reply:
Yep, just remember to update the docs about how to update the versions :)

Reply:
@matpfernandes, I've added to `__init__.py` file the version of the package. Could you please see if that's ok for you?

Reply:
In the future, I'll add to `README` how to update the version.